Essentials/src/config.yml

```diff
@@ -692,6 +692,16 @@ max-money: 10000000000000
 # Setting this to 0, will disable overdrafts/loans completely.  Users need 'essentials.eco.loan' perm to go below 0.
 min-money: -10000
 
+# Enable this to make sell signs automatically adjust their price according to worth.yml
+economy-update-sell-sign-to-worth: false
+
+# Enable this to make buy signs automatically adjust their price according to worth.yml
+economy-update-buy-sign-to-worth: false
```

Comment:
Not a fan of the descriptions here, 'automatically' implies that all signs would update upon reloading

Reply:
I'll change this